app/src/main/java/com/example/caregiver/Dashboard.java

```diff
@@ -35,7 +47,11 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
                 fragment = new HomeFragment();
                 break;
             case R.id.task:
-                fragment = new TaskFragment();
+                if (role.equals("caregivee")){
```

Comment:
All this does is create 2 different fragments based on whether the user is caregiver or caregivee